docs/source/pythonapi/examples/mgxs-part-ii.ipynb

```diff
@@ -1741,84 +1973,73 @@
    "source": [
     "It is often insightful to generate visual depictions of multi-group cross sections. There are many different types of plots which may be useful for multi-group cross section visualization, only a few of which will be shown here for enrichment and inspiration.\n",
     "\n",
-    "One particularly useful visualization is a comparison of the continuous-energy and multi-group cross sections for a particular nuclide and reaction type. We illustrate one option for generating such plots with the use of the `openmc.data` module to parse continuous-energy cross sections from an openly available ACE cross section library distributed by NNDC. First, we instantiate a `openmc.data.IncidentNeutron` object for U-235 as follows."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 30,
-   "metadata": {
-    "collapsed": false
-   },
-   "outputs": [],
-   "source": [
-    "# Parse ACE data into memory\n",
-    "u235 = openmc.data.IncidentNeutron.from_ace('../../../../scripts/nndc/293.6K/U_235_293.6K.ace')\n",
+    "One particularly useful visualization is a comparison of the continuous-energy and multi-group cross sections for a particular nuclide and reaction type. We illustrate one option for generating such plots with the use of the `openmc.plotter` module to plot continuous-energy cross sections from the openly available cross section library distributed by NNDC.\n",
     "\n",
-    "# Extract the continuous-energy U-235 fission cross section data\n",
-    "fission = u235[18]"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Now, we use [`matplotlib`](http://matplotlib.org/) and [`seaborn`](http://stanford.edu/~mwaskom/software/seaborn/) to plot the continuous-energy and multi-group cross sections on a single plot."
+    "There is a simpler way to plot the MGXS data (using the same interface as is used for the continuous-energy data), however this example series has not yet introduced the pre-requisite information and so we will do this manually here."
```

Comment:
I think it's useful to show this manual method because it makes it really clear what you get out of the `get_xs` method.  It would probably improve this comment to mention `openmc.plot_xs` by name though.